minpy/tape.py

```diff
@@ -132,6 +132,14 @@ def add_gradient(arr, grad):
 
         add_gradient(arr, grad)
 
+    def get_result_grad(self, result, primitive_type):
```

Comment:
No need to make it a member method. Shouldn't be called anywhere else. Can you move it back where it was?